composer.json

```diff
@@ -97,9 +103,11 @@
       "vendor/bin/phpcbf --standard=phpcs.xml ./src"
     ],
     "phpcpd": [
-      "vendor/bin/phpcpd  ./src",
-      "vendor/bin/phpcpd  ./tests"
+      "vendor/bin/phpcpd  ./src"
     ],
+    "jest": "yarn jest",
```

Comment:
Why do you use the composer.json and not the package.json for these tasks?

Reply:
Because we already use composer as task manager, this is only reference for @test run command of composer